OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -458,7 +461,7 @@ computeInitialFiberEquilibrium(SimTK::State& s) const
             case 2: //maximum number of iterations reached
             {
                 setActuation(s, 0.0);
-                setFiberLength(s,penMdl.getOptimalFiberLength());
+                setFiberLength(s, get_optimal_fiber_length());
```

Comment:
same as above. The case structure is probably unnecessary as well. It either converged or failed, in which case the `default` case is not materially different from this case. Perhaps for another pass.